ruby/README.md

```diff
@@ -29,8 +29,75 @@ Flexport/InclusiveCode:
   DisableAutoCorrect: false
 ```
 
-You can run the cop on your entire codebase with `rubocop --only Flexport/InclusiveCode`. 
+You can run the cop on your entire codebase with `rubocop --only Flexport/InclusiveCode`.
 
 You can run the cop on a specific file with `rubocop --only Flexport/InclusiveCode file_path`.
 
 If you want to add inline `rubocop:disable` or `rubocop:todo` comments on all offenses, set `DisableAutoCorrect: true` in your .rubocop.yml, and run `rubocop --only Flexport/InclusiveCode --auto-correct --disable-uncorrectable`.
+
```

Comment:
@helenuria, here's my attempt at expanding the README to include more details about configuration.

What do you think?

I think it would be good to indicate which key/value pairs are required and which are optional. Could you help me by providing that information?

Some specific questions that came up for me when writing this:
- is it required to provide suggestions?
- if no suggestions are provided, will autocorrect substitute an empty string for any offenses it finds?

Reply:
Adding specs for my questions... It appears that:
- it is not required to provide suggestions
- if no suggestions are provided, auto-correct would work as is likely desired (i.e. it would make no change)

Currently we do require a value for the `allowed` key. It would probably preferable to accept a flagged terms config having no `allowed` key.

I'll update specs to reflect some of these findings, and then take a look at including any relevant info in the readme.

Reply:
I agree itd be preferable to accept flagged terms config having no allowed key.